lib/Jsrt/JsrtCommonExports.inc

```diff
@@ -5,6 +5,7 @@
     JsAddRef
     JsRelease
     JsCreateContext
+    JsGetContextUrl
```

Comment:
This is the only way for a ChakraCore host to get the url I guess?  Probably want to run this by @curtisman if haven't already.